dbms/src/TestUtils/mockExecutor.cpp

```diff
@@ -256,4 +290,8 @@ DAGRequestBuilder MockDAGRequestContext::scan(String db_name, String table_name)
     return DAGRequestBuilder(index).mockTable({db_name, table_name}, mock_tables[db_name + "." + table_name]);
 }
 
+DAGRequestBuilder MockDAGRequestContext::receive(String db_name, String table_name)
```

Comment:
It's a little weird to use `db_name` and `table_name`...

Reply:
I will save the relationship between sender and receiver.